package.json

```diff
@@ -125,7 +125,7 @@
     "yargs": "17.5.1"
   },
   "peerDependencies": {
-    "@nebula.js/stardust": "3.x"
+    "@nebula.js/stardust": ">=3.0.3"
```

Comment:
so it will not support 3.0.2 anymore? seems the previous one could do.

Reply:
Ah, ok, because the mashup issue, you said before.

Reply:
yes :)